UNITTESTS/stubs/CMakeLists.txt

```diff
@@ -101,7 +91,6 @@ target_include_directories(mbed-stubs-headers
 )
 
 add_subdirectory(connectivity)
-add_subdirectory(events)
 add_subdirectory(platform)
 add_subdirectory(storage)
 
```

Comment:
(Referring to the lines below)
Shall we remove mbed-stubs-events from mbed-stubs, and let tests explicitly link mbed-stubs-events? Similar to what's been done to the headers - only link what is used.
Once all component stubs have been refactors, it'd be nice to have no global mbed-stubs but only component stubs eventually.

Reply:
Oh sorry this duplicates my comment from a few hours ago - I was re-checking the platforms PR (#14773) and just added the same comment to all other PRs 😅

Reply:
I end up with a build issue, and we will do further cleanup or removing the complete mbed-os/UNITTESTS directory after we moved everything out